src/server/state.rs

```diff
@@ -40,6 +42,39 @@ pub type StateRef = WrappedRcRefCell<State>;
         })
 }*/
 
+fn cancel_tasks_from_callback(
+    state_ref: &StateRef,
+    tako_ref: &TakoServer,
+    job_id: JobId,
+    tasks: Vec<TakoTaskId>,
+) {
+    if tasks.is_empty() {
+        return;
+    }
+    let tako_ref = tako_ref.clone();
+    let state_ref = state_ref.clone();
+    tokio::task::spawn_local(async move {
+        let message = FromGatewayMessage::CancelTasks(CancelTasks { tasks });
```

Comment:
`rpc_call!`?

Reply:
``rpc_call`` is for client messages, this is a hq <-> tako message.

Reply:
Ah. So, should we introduce `tako_call!`? :)

Reply:
Good idea, but for another PR:)